executor/join_test.go

```diff
@@ -953,7 +954,7 @@ func (s *testSuite2) TestHashJoin(c *C) {
 	row := result.Rows()
 	c.Assert(len(row), Equals, 5)
 	outerExecInfo := row[1][4].(string)
-	c.Assert(outerExecInfo[len(outerExecInfo)-1:], Equals, "1")
+	c.Assert(outerExecInfo[len(outerExecInfo)-1:], Equals, "5") // FIX ME after distsql follow initChunkSize
```

Comment:
If distsql uses initChunkSize, this test will fail?

Reply:
if distsql use initChunkSize will got right 1

Reply:
IMHO, we need to make the comments easily to be understood by the outsider.
The comments introduced by this PR are somewhat ugly.